src/remote_tables.rs

```diff
@@ -103,13 +105,14 @@ impl TableProvider for RemoteTable {
         // Scope down the schema and query column specifiers if a projection is specified
         let mut schema = self.schema.deref().clone();
         let mut columns = "*".to_string();
+
         if let Some(indices) = projection {
             schema = schema.project(indices)?;
             columns = schema
                 .fields()
                 .iter()
-                .map(|f| f.name().as_str())
-                .collect::<Vec<&str>>()
+                .map(|f| format!("\"{}\"", f.name()))
```

Comment:
It should be something like `f.name().replace("\"", "\"\"")` (at least in PG, you can escape double quotes in identifier names with double-double quotes, though in MySQL identifiers are surrounded by backticks, so this won't be platform-independent.